components/molecules/m-ideal-issuers.vue

```diff
@@ -53,10 +53,18 @@ export default {
       })
     },
     isIdeal () {
-      return (this.paymentDetails.paymentMethod === 'ideal')
```

Comment:
@cowoder We get's unexpected a paymentMethod value after copy the shipping address to billing.